src/services/dashboards/dashboards.svc.coffee

```diff
@@ -157,6 +167,8 @@ angular
       widgetsTemplatesPromises.push $http.get(widgetsTemplatesUrl()).then(
         (response) ->
           for widgetTemplate in response.data.widgets
+            if _.get(widgetTemplate, 'metadata.hist_parameters.from') && _.get(widgetTemplate, 'metadata.hist_parameters.to')
```

Comment:
Same here. Plus, remember the `metadata` objects will be sent to the Impac! API. You can't assume the API will accept any date format...